incuna_test_utils/testcases/urls.py

```diff
@@ -23,11 +23,10 @@ def assert_url_matches_view(self, view, expected_url, url_name,
         self.assertEqual(reversed_url, expected_url)
 
         resolved_view = resolve(expected_url, urlconf=urlconf).func
-
-        if hasattr(resolved_view, 'cls'):
-            self.assertEqual(resolved_view.cls, view)
-        else:
-            self.assertEqual(resolved_view.__name__, view.__name__)
+        self.assertEqual(
+            (resolved_view.__module__, resolved_view.__name__),
+            (view.__module__, view.__name__),
```

Comment:
Looks like a nice feature :)
Is it a good idea to add a `msg` in case of failure? `msg` could be a default or could be user defined?